### PR TITLE
Edit docs

### LINKED ISSUE #2679: CMS: Docs texts to proofread
We have new documentation in
- [x] http://opendata-dev.web.cern.ch/docs/cms-mc-production-overview
- [x] http://opendata-dev.web.cern.ch/docs/cms-guide-event-production
- [x] http://opendata-dev.web.cern.ch/docs/cms-guide-docker
- [x] http://opendata-dev.web.cern.ch/docs/simulated-dataset-categories

@RaoOfPhysics I did not pick up anything in them, but it would be very nice, though, if you could read them through and make a PR for eventual corrections. Thanks!

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
Closes #2679 

Sorry, I forgot to include this in the commit log.

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/cms-guide-event-production/cms-guide-event-production.md [hunk 1]
```diff
@@ -1,81 +1,79 @@
...
+- There is a difference between processing MC events and real events. For the former, the `cmsDriver.py` tool helps in putting together configuration files for the generation of such events, the simulation of the interaction with the detector, and its possible reconstruction. For the latter, there will be little need of doing anything because most of the Open Data released by CMS is already *reconstructed*, ready to be analysed.  However, for a set of small *raw* samples, one might need to use the `cmsDriver.py` tool to put together configuration files that can perform the full reconstruction.
+- These instructions focus on MC-event production. The idea is to use the `cmsDriver.py` to be able to build *any* configuration file out of the large collection of CMSSW pre-fabricated configuration application fragments. Thus, in most cases a user will only need to know how find the right components, to compose them together, and to execute.
```
Comment (caredg) [inline cernopendata/modules/fixtures/data/docs/cms-guide-event-production/cms-guide-event-production.md]: there is a typo "how find" --> "how to find"

File: cernopendata/modules/fixtures/data/docs/cms-guide-event-production/cms-guide-event-production.md [hunk 2]
```diff
@@ -1,81 +1,79 @@
...
+- When it comes to the configuration input files, one needs to remember that the event processing usually happens in many steps.  This means that the `cmsDriver.py` might need to be used several times to acquire the configuration files that are needed for each step. One step in the event-processing chain may create event data (*edm::Event*) that will serve as an input to the subsequent steps. Thus, it is important to properly order the sequence of steps for execution.
+- One also need to know that some software components may need other *helper* software, so those services and modules also need to be present in the configuration file. However, most users will not need to worry about such details as there are utilities in CMSSW that will ensure that all service software is included, and the sequence of steps in the event processing chain is correct.
```
Comment (caredg) [inline cernopendata/modules/fixtures/data/docs/cms-guide-event-production/cms-guide-event-production.md]: "One also need"--> "One also needs"

**Comment by RaoOfPhysics** [comment]
Thanks, @caredg, now fixed.

File: cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md [hunk 3]
```diff
@@ -27,29 +26,27 @@ Digest: sha256:6b9a12992ba088a168b87df98a841d3c56dede326684f5551368fd359acfb43c
...
 cmsusr@eb9ecf54fd2a ~/CMSSW_5_3_32/src $
-`
+```
 
 ### Instructions for 2010 data
 
 Once Docker is installed, you can fetch a CMSSW image and create and start a container using the `docker run` command:
 
-`
+```bash
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md]: If you put the prompt, i.e. the block is a shell session example, then it had better be `console`.  If the block contains `shell` or `bash` script code, then it should not contain prompt, as it should be copy-paste-runnable.

Examples: 

```
'''console
$ whoami
$ date
'''

'''sh
whoami
date
'''
```

Note that the former can contain the command outputs:

```
'''console
$ whoami
root
$ date
Tue 25 Jun 16:16:50 CEST 2019
'''
```

while the second (`sh`) cannot.

(Note that this remark also applies to other blocks in this PR.)

Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md]: Ugh, I thought I had removed this everywhere. Thanks for the pointers, I'll fix it everywhere.

**Comment by RaoOfPhysics** [comment]
Thanks, @tiborsimko, should now be correct.

File: cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md [hunk 4]
```diff
@@ -27,29 +26,27 @@ Digest: sha256:6b9a12992ba088a168b87df98a841d3c56dede326684f5551368fd359acfb43c
...
 ### Instructions for 2010 data
 
 Once Docker is installed, you can fetch a CMSSW image and create and start a container using the `docker run` command:
 
-`
-$ docker run --name opendata-2010 -it clelange/cmssw:4_2_8 /bin/bash
-`
+```sh
+docker run --name opendata-2010 -it clelange/cmssw:4_2_8 /bin/bash
+````
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md]: Should be three backticks only

Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md]: Otherwise looks good. I'll fix this during rebase and merge

Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md]: Ack! Sorry! Thanks for spotting and for fixing. :)

**Pull request merged**